autospec/git.py

```diff
@@ -60,7 +60,6 @@ def commit_to_git(path):
     call("bash -c 'shopt -s failglob; git add -f *.sha256'", check=False, stderr=subprocess.DEVNULL, cwd=path)
     call("bash -c 'shopt -s failglob; git add -f *.sign'", check=False, stderr=subprocess.DEVNULL, cwd=path)
     call("bash -c 'shopt -s failglob; git add -f *.pkey'", check=False, stderr=subprocess.DEVNULL, cwd=path)
-    call("bash -c 'shopt -s failglob; git add -f *_extras'", check=False, stderr=subprocess.DEVNULL, cwd=path)
```

Comment:
Don't we still want to commit any files that match `*_extras`? Though it needs an extension to `*_extras*` now.

Reply:
This is something that is handled via read_conf_file, when foo_extras and foo_extras_requires is read they are added to the set of config_files that git manages.

Reply:
Ah, yes, I see that now, thanks.